db/migrate/20230221082231_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] Please 🙏🏽 add` t.integer :likes_counter` and `t.integer :comments_counter` columns on Post table to register the number of likes and comments.
- [ ] Kindly add the references to the user table 👉🏽 `t.references :author, foreign_key: { to_table: 'users' }`. It represents the relationship between the user and the posts tables.